From: Prarit Bhargava <prarit@redhat.com>
Subject: [RHEL5.1 PATCH]: Add additional device IDs for SB700
Date: Mon, 23 Jul 2007 09:27:30 -0400
Bugzilla: 248109
Message-Id: <20070723132730.22052.82349.sendpatchset@prarit.boston.redhat.com>
Changelog: [ata]: Add additional device IDs for SB700


Add additional device IDs for SB700.

Backport of

http://git.kernel.org/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=badc2341579511a247f5993865aa68379e283c5c;hp=203ef6c456ad70e660cca67921d3d872b13aa516

Successfully compile tested by me (no HW in Westford, however this is
upstream and I'm assuming that AMD/ATI have done appropriate testing).

diff -urNp linux-2.6.18.x86_64.orig/drivers/ata/ahci.c linux-2.6.18.x86_64/drivers/ata/ahci.c
--- linux-2.6.18.x86_64.orig/drivers/ata/ahci.c	2007-07-13 20:28:21.000000000 -0400
+++ linux-2.6.18.x86_64/drivers/ata/ahci.c	2007-07-19 15:31:53.000000000 -0400
@@ -399,7 +399,10 @@ static const struct pci_device_id ahci_p
 
 	/* ATI */
 	{ PCI_VDEVICE(ATI, 0x4380), board_ahci_sb600 }, /* ATI SB600 */
-	{ PCI_VDEVICE(ATI, 0x4390), board_ahci_sb600 }, /* ATI SB700 */
+	{ PCI_VDEVICE(ATI, 0x4390), board_ahci_sb600 }, /* ATI SB700 IDE */
+	{ PCI_VDEVICE(ATI, 0x4391), board_ahci_sb600 }, /* ATI SB700 AHCI */
+	{ PCI_VDEVICE(ATI, 0x4392), board_ahci_sb600 }, /* ATI SB700 nraid5 */
+	{ PCI_VDEVICE(ATI, 0x4393), board_ahci_sb600 }, /* ATI SB700 raid5 */
 
 	/* VIA */
 	{ PCI_VDEVICE(VIA, 0x3349), board_ahci_vt8251 }, /* VIA VT8251 */

